Documentation/ceph-nfs-crd.md

```diff
@@ -72,55 +82,395 @@ spec:
+editor. The file should look similar to below when the edit is finished.
+```
+$ cat /tmp/<export>.conf
+EXPORT {
+    export_id = 1;
+    path = "/";
+    pseudo = "/test";
+    access_type = "RW";
+    squash = "no_root_squash";
+    protocols = 4;
+    transports = "TCP";
+    FSAL {
+        name = "CEPH";
+        filesystem = "myfs";
+    }
+
+}
+```
+
+Now that the old user and access key are removed, import the export. There should be no errors, but
+if there are, follow the error message instructions to proceed.
+```sh
+ceph nfs export apply <cephnfs-name> -i /tmp/<export>.conf
+```
+
+### Step 5
+Once all exports have been migrated for the current CephNFS, it is good to verify the exports. Use
+`ceph nfs export ls <cephnfs-name>` to list all exports (identified by the pseudo path), and use
+`ceph nfs export info <cephnfs-name> <export-pseudo>` to inspect the configuration. An export
+configuration may look something like below.
+
+```
+$ ceph nfs export info my-nfs "/test"
+{
+  "export_id": 1,
+  "path": "/",
+  "cluster_id": "my-nfs",
+  "pseudo": "/test",
+  "access_type": "RW",
+  "squash": "no_root_squash",
+  "security_label": true,
+  "protocols": [
+    4
+  ],
+  "transports": [
+    "TCP"
+  ],
+  "fsal": {
+    "name": "CEPH",
+    "user_id": "nfs.my-nfs.1",
+    "fs_name": "myfs"
+  },
+  "clients": []
+}
+```
+
+### Step 6
+Repeat these [steps](#step-1) for each other CephNFS.
+
+Clean up all `<export>.conf` files before moving onto subsequent CephNFSes to avoid confusion.
+```
+rm -f /tmp/export-*.conf
+```
+
+### Wrap-up
+Once you are finished migrating all CephNFSes, the migration is complete. If you wish to use the
+Ceph dashboard to manage exports, you should now be able to find them all listed there.
+
+If you are done managing NFS exports via the CLI and don't need the Ceph orchestrator module enabled
+for anything else, it may be preferable to disable the Rook and NFS mgr modules to free up a small
+amount of RAM in the Ceph mgr Pod.
+```console
+ceph mgr module disable nfs
+ceph mgr module disable rook
+```
 
-When a server is started, it will create the included object if it does not already exist. It is possible to prepopulate the included objects prior to starting the server. The format for these objects is documented in the [NFS Ganesha](https://github.com/nfs-ganesha/nfs-ganesha/wiki) project.
 
 ## Scaling the active server count
+It is possible to scale the size of the cluster up or down by modifying the `spec.server.active`
+field. Scaling the cluster size up can be done at will. Once the new server comes up, clients can be
+assigned to it immediately.
+
+The CRD always eliminates the highest index servers first, in reverse order from how they were
+started. Scaling down the cluster requires that clients be migrated from servers that will be
+eliminated to others. That process is currently a manual one and should be performed before reducing
+the size of the cluster.
+
```

Comment:
unecessary blank line?

Reply:
I've been keeping two newlines between the major lvl-2 headers to help with navigating the raw .md file a bit easier.